packages/loot-core/src/shared/errors.ts

```diff
@@ -46,17 +60,16 @@ export function getDownloadError({ reason, meta, fileName }) {
       );
 
     case 'out-of-sync-migrations':
-      return (
-        'This budget cannot be loaded with this version of the app. ' +
-        'Make sure the app is up-to-date.'
+      return t(
+        'This budget cannot be loaded with this version of the app. Make sure the app is up-to-date.',
       );
 
     default:
       const info = meta && meta.fileId ? `, fileId: ${meta.fileId}` : '';
       return (
-        'Something went wrong trying to download that file, sorry! ' +
-        'Visit https://actualbudget.org/contact/ for support. ' +
-        `(reason: ${reason}${info})`
+        t('Something went wrong trying to download that file, sorry! ') +
+        t('Visit https://actualbudget.org/contact/ for support. ') +
+        `(${t('reason:')} ${reason}${info})`
```

Comment:
This should be one big string translation rather than 3 small ones - it makes it easier for translators to understand what it's saying